### DISCUSSION TITLE
main search in powergrid was not working when using joining in multiple table

### DISCUSSION THREAD

**Discussion by srikanthmuviereck** [Power-Components/livewire-powergrid#763] [BUG]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

Yes - I didn't work.

### Is there an error in the console?

No

### Theme

Bootstrap

### Describe the bug.

main search in powergrid was not working when using joining in multiple table

### To Reproduce...

First click on "FOO" then....

### Extra information

```markdown
<?php
 //...
```

**Comment by luanfreitasdev** [comment]
Hello!, please provide more details so we can identify the problem. If possible put a screenshot of the error and how you set up the table
